girder/models/item.py

```diff
@@ -540,3 +540,83 @@ def checkConsistency(self, stage, progress=None):
                     progress.update(increment=1, message='Checking items (%d '
                                     'left)' % itemsLeft)
             return numItems, itemsCorrected
+
+    def _mergeMetadataHelper(self, oldDoc, newDoc, type='item'):
+        # metadata
+        if 'meta' in oldDoc:
+            if 'meta' in newDoc:
+                newDoc['meta'] = dict(
+                    newDoc['meta'].items() + oldDoc['meta'].items())
```

Comment:
Really unimportant, but another option for lines 548-9 would just be `newDoc['meta'].update(oldDoc['meta'])`.